infra/aws/terraform/service-quotas/boskos/README.md

```diff
@@ -0,0 +1,27 @@
+# Service Quotas for Boskos Accounts
+
+This directory contains Terraform configs to apply service quotas in boskos
+accounts. Currently, these configs are only applying
```

Comment:
Maybe let's be specific and mention that those are AWS accounts?

Reply:
It's in `infra/aws` directory, so I thought that's kinda implicit 😄

Reply:
Fixed